style.css

```diff
@@ -4972,6 +4974,16 @@ div#content .main p {
     grid-gap: 40px;
     padding: 60px 0;
 }
+
+.archivedStatus{
+	border:5px solid var(--archivedStatusColor);
```

Comment:
Instead of `border` with the semi-opaque shorthand, which I consider a CSS anti-pattern, use specific `border-width`, `border-style`, and `border-color`. And Generalize the width and style so they can just apply to `.projectStatus` things.